ajax/libs/jquery.flipster/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jquery.flipster",
+  "version": "1.1.2",
+  "homepage": "http://brokensquare.com/Code/jquery-flipster/demo/",
+  "description": "Responsive coverflow-style carousel plugin for jQuery",
+  "filename": "jquery.flipster.min.js",
+  "author": "Adrien Delessert <a@drien.com>",
+  "keywords": [
+    "coverflow",
+    "jquery",
+    "carousel",
+    "flipster",
+    "jquery-plugin",
+    "ecosystem:jquery"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/drien/jquery-flipster.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:drien/jquery-flipster.git",
```

Comment:
Please use `git://github.com/drien/jquery-flipster.git`, be care for ':'. Thanks!

Reply:
@maruilian11 
I made the requested changes. Thank you!